### PR TITLE
Aula1

### PR DESCRIPTION
**Pull request by PedroDeSanti**
PR da aula 5 (sim o nome é conflitante kk)

### CODE REVIEW

**Comment by PedroDeSanti** [comment]
@all-contributors please add @PedroDeSanti for Code, Reviewed Pull Requests, Documentation
@all-contributors please add @jvtdegelo for Code, Reviewed Pull Requests, Documentation
@all-contributors please add @GabZamba for Code, Reviewed Pull Requests, Documentation

**Comment by PedroDeSanti** [comment]
https://github.com/all-contributors please add @PedroDeSanti for Code, Reviewed Pull Requests, Documentation

**Comment by PedroDeSanti** [comment]
@all-contributors please add @PedroDeSanti for Code, Reviewed Pull Requests, Documentation

**Comment by PedroDeSanti** [comment]
@all-contributors please add @jvtdegelo for Code, Reviewed Pull Requests, Documentation

**Comment by PedroDeSanti** [comment]
@all-contributors please add @GabZamba for Code, Reviewed Pull Requests, Documentation

**Comment by PedroDeSanti** [comment]
@all-contributors please add @jvtdegelo for Code, Reviewed Pull Requests, Documentation on branch aula1

**Comment by PedroDeSanti** [comment]
@all-contributors please add @GabZamba for Code, Reviewed Pull Requests, Documentation on branch aula1

File: FlightOps/book/views.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from django.shortcuts import render
+
+#Create your views here.
+def bookview(request):
+    return render(request, "FIRST.html")
```
Comment (PedroDeSanti) [inline FlightOps/book/views.py]:
```suggestion
    return render(request, "FIRST.html")
    
```

File: FlightOps/template/FIRST.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  PCS3642 - Laboratório de Engenharia de Software I, também conhecida como a MELHOR matéria da USP 🥇
+</body>
+</html>
```
Comment (PedroDeSanti) [inline FlightOps/template/FIRST.html]:
```suggestion
</html>

```

Comment (PedroDeSanti) [reply FlightOps/template/FIRST.html]: Bom deixar espaço no final

File: FlightOps/book/views.py (hunk 1 above)
Comment (PedroDeSanti) [reply FlightOps/book/views.py]: espacinho no final

File: FlightOps/manage.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+"""Django's command-line utility for administrative tasks."""
+import os
+import sys
+
+
+def main():
```
Comment (PedroDeSanti) [inline FlightOps/manage.py]:
```suggestion
import sys

def main():
```

espaco desnecessario

File: FlightOps/book/apps.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class BookConfig(AppConfig):
```
Comment (PedroDeSanti) [inline FlightOps/book/apps.py]:
```suggestion
from django.apps import AppConfig

class BookConfig(AppConfig):
```

Espaço desnecessario

**Review by PedroDeSanti — COMMENTED** [review]
Parece check

**Review by GabZamba — APPROVED** [review]
lgtm

**Review by jvtdegelo — APPROVED** [review]
lindo demais

**Pull request merged**